<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowNewCompany</class>
 <widget class="QMainWindow" name="WindowNewCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>New Company Creation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewCompanyDone">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Company</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_companyName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_newUsername">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_verifyPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_verify_password">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_new_username">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_lastName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_SSN">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SSN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ssn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="placeholderText">
     <string>###-##-####</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_last_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_first_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ediImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/new/prefix1/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_employee_id">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>9</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_employeeID">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Employee ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label_title</zorder>
   <zorder>pushButtonNewCompanyDone</zorder>
   <zorder>label_companyName</zorder>
   <zorder>label_newUsername</zorder>
   <zorder>label_password</zorder>
   <zorder>label_verifyPassword</zorder>
   <zorder>lineEdit_company_name</zorder>
   <zorder>lineEdit_verify_password</zorder>
   <zorder>lineEdit_password</zorder>
   <zorder>label_6</zorder>
   <zorder>label_lastName</zorder>
   <zorder>label_SSN</zorder>
   <zorder>lineEdit_ssn</zorder>
   <zorder>lineEdit_last_name</zorder>
   <zorder>lineEdit_first_name</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_ediImage</zorder>
   <zorder>lineEdit_new_username</zorder>
   <zorder>lineEdit_employee_id</zorder>
   <zorder>label_employeeID</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_company_name</tabstop>
  <tabstop>lineEdit_new_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_verify_password</tabstop>
  <tabstop>lineEdit_first_name</tabstop>
  <tabstop>lineEdit_last_name</tabstop>
  <tabstop>lineEdit_employee_id</tabstop>
  <tabstop>lineEdit_ssn</tabstop>
  <tabstop>pushButtonNewCompanyDone</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
